django/test/runner.py

```diff
@@ -101,6 +101,43 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
+class TestCaseDTO:
```

Comment:
What does DTO stand for? I think it'd be better to use a spelled out name for this class

Reply:
https://en.wikipedia.org/wiki/Data_transfer_object

Happy to rename for clarity